Add proxmox community-scripts (#235)
homarr-labs/documentation: docs/getting-started/installation/proxmox.mdx

# Proxmox

This guide covers how to install homarr inside of a linux-container in ProxmoxVE, using a fully automated approach.

# What is community-scripts?

https://community-scripts.github.io/ProxmoxVE/

> We are a community-driven initiative that simplifies the setup of Proxmox Virtual Environment (VE).
> With hundreds of scripts to help you manage your Proxmox VE environment. Whether you're a seasoned user or a newcomer, we've got you covered.
> 

Reviewer: Is is not clear how the scripts community is related to this script. Therefore, I suggest the following:
```suggestion
This installation method is provided by the Proxmox Community Scripts. We are a community-driven initiative that simplifies the setup of Proxmox Virtual Environment (VE).
With hundreds of scripts to help you manage your Proxmox VE environment. Whether you're a seasoned user or a newcomer, we've got you covered.

```

# Installation

> It's as simple as pasting the following snippet into your Proxmox Node's console

Reviewer: ```suggestion
It's as simple as pasting the following snippet into your Proxmox node's console
```

```bash
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/homarr.sh)"
```

> :::warning
> Never trust any script directly injected from the web, without prior looking through it or searching the web for other's opinions.

Reviewer: End the note after this sentence:
``Never trust any script directly injected from the web, without prior looking through it or searching the web for other's opinions.``
![image](https://github.com/user-attachments/assets/7f75f2f0-c0be-49be-8606-e4c4c828c24f)
```suggestion
:::warning

Never trust any script directly injected from the web, without prior looking through it or searching the web for other's opinions.

:::
```

> After executing this command, you will get asked, if you want to accept defaults or would like to specify things like IP-Adress, DNS etc, then choose advanced.

Reviewer: Instead of glossing over those options, why not document them here and what values can be chosen or how they can be chosen (e.g. by keyboard?)

> The installation part takes a moment, but you there is a nice progress-spinner.

Reviewer: ```suggestion
After confirming the installation, the script will install the required dependencies, which may take a few minutes. Do not interrupt the installation.
```

After you've installed Homarr, you can find a new LXC "homarr".

# Updating

Updating is as simple as executing 

```bash
update
```

or running the script again but this time inside the container's shell:

```bash
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/homarr.sh)"
```

# Troubleshooting

If something does not work after an update, please first try rebooting the LXC or the homarr Service with 

```bash
systemctl restart homarr
```

# Getting help

> The official support of community-scripts is located [here](https://discord.gg/2wvnMDgdnU) if you feel like there is an issue with the script, feel free ot open a github issue [here](https://github.com/community-scripts/ProxmoxVE/issues/new/choose)
> 
> If you feel like the issue is at homarr side, please ask on the homarr Discord or Github instead.

Reviewer: ```suggestion
- If you are looking for help with the script, updating the LXC or troubleshooting the installer, please get in contact with the support of the community scripts. They provide support on their [Discord](https://discord.gg/2wvnMDgdnU). If you found vulnerabilities, bugs or have a request for the script, [create a new issue here](https://github.com/community-scripts/ProxmoxVE/issues/new/choose).
- If you are looking to report a bug, vulnerability or request a new feature, use https://github.com/homarr-labs/homarr . You can ask for help on our Discord.
```

# How to find logs

> Homarr runs under a system service called `homarr`, so getting the logs is a simple as using

Reviewer: This must be executed in the VM, no?

```bash
systemctl status homarr
```

> As homarr logs really a log, it sometimes feels like the log is endless till you get the bottom, so a command like this only shows the latest 100 lines:

Reviewer: ```suggestion
As Homarr logs really a log, it sometimes feels like the log is endless till you get the bottom, so a command like this only shows the latest 100 lines:
```
